v2/setup/marathon.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+source /etc/environment
+
+HOMEDIR=$(eval echo "~`whoami`")
+
+sudo docker run --rm \
+    -v ${HOMEDIR}:/data/ behance/docker-aws-s3-downloader \
+     us-east-1 $CONTROL_TIER_S3SECURE_BUCKET .marathon
+
+
+etcdctl set /marathon_user $MARATHON_USER
```

Comment:
Also, I would assume that you're reading these from the `.marathon` file above.
You might want to actually read the file. It's not going to be in the environment.